File: v2/java/src/main/java/vestorly/client/model/Source.java
package vestorly.client.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Source  {
  
  private String id = null;
  private String name = null;
  private String url = null;
  private String logoUrl = null;
  private Boolean enabled = null;
  private Boolean customRssFeed = null;
  private String rssPublisher = null;

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("_id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("name")
  public String getName() {
    return name;
  }
  public void setName(String name) {
    this.name = name;
  }

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("url")
  public String getUrl() {
    return url;
  }
  public void setUrl(String url) {
    this.url = url;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("logo_url")
  public String getLogoUrl() {
    return logoUrl;
  }
  public void setLogoUrl(String logoUrl) {
    this.logoUrl = logoUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("enabled")
  public Boolean getEnabled() {
    return enabled;
  }
  public void setEnabled(Boolean enabled) {
    this.enabled = enabled;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("custom_rss_feed")
  public Boolean getCustomRssFeed() {
    return customRssFeed;
  }
  public void setCustomRssFeed(Boolean customRssFeed) {
    this.customRssFeed = customRssFeed;
  }

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  @JsonProperty("rss_publisher")
  public String getRssPublisher() {
    return rssPublisher;
  }
  public void setRssPublisher(String rssPublisher) {
    this.rssPublisher = rssPublisher;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Source {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  name: ").append(name).append("\n");
    sb.append("  url: ").append(url).append("\n");
    sb.append("  logoUrl: ").append(logoUrl).append("\n");
    sb.append("  enabled: ").append(enabled).append("\n");
    sb.append("  customRssFeed: ").append(customRssFeed).append("\n");
    sb.append("  rssPublisher: ").append(rssPublisher).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/Source.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class Source {
    
    
    [DataMember(Name="_id", EmitDefaultValue=false)]
    public string id { get; set; }

    
    
    [DataMember(Name="name", EmitDefaultValue=false)]
    public string name { get; set; }

    
    
    [DataMember(Name="url", EmitDefaultValue=false)]
    public string url { get; set; }

    
    
    [DataMember(Name="logo_url", EmitDefaultValue=false)]
    public string logoUrl { get; set; }

    
    
    [DataMember(Name="enabled", EmitDefaultValue=false)]
    public bool? enabled { get; set; }

    
    
    [DataMember(Name="custom_rss_feed", EmitDefaultValue=false)]
    public bool? customRssFeed { get; set; }

    
    
    [DataMember(Name="rss_publisher", EmitDefaultValue=false)]
    public string rssPublisher { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Source {\n");
      
      sb.Append("  id: ").Append(id).Append("\n");
      
      sb.Append("  name: ").Append(name).Append("\n");
      
      sb.Append("  url: ").Append(url).Append("\n");
      
      sb.Append("  logoUrl: ").Append(logoUrl).Append("\n");
      
      sb.Append("  enabled: ").Append(enabled).Append("\n");
      
      sb.Append("  customRssFeed: ").Append(customRssFeed).Append("\n");
      
      sb.Append("  rssPublisher: ").Append(rssPublisher).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
